app/src/main/java/com/example/justjet/Ui/LoginFragment.java

```diff
@@ -0,0 +1,125 @@
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.example.justjet.R;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+import java.util.Objects;
+
+
+public class LoginFragment extends Fragment implements View.OnClickListener {
+    public LoginFragment() {
+        // Required empty public constructor
+    }
+    FirebaseAuth firebaseAuth;
+    private FirebaseUser mUser;
+    private static EditText emailid, password;
+    private static Button loginButton;
+    private static TextView signUp;
+    private static LinearLayout loginLayout;
+
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+                          @Nullable Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_login, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        final Bundle bundle = new Bundle();
+        bundle.putBoolean("test_boolean", true);
+
+        final NavController navController = Navigation.findNavController(requireActivity(), R.id.my_nav_host_fragment);
+        firebaseAuth = FirebaseAuth.getInstance();
+            mUser = FirebaseAuth.getInstance().getCurrentUser();
+        emailid = (EditText) view.findViewById(R.id.Lemail);
+        password = (EditText) view.findViewById(R.id.Lpass);
+        loginButton = (Button) view.findViewById(R.id.login);
+        signUp = (TextView) view.findViewById(R.id.sign);
+        loginLayout = (LinearLayout) view.findViewById(R.id.loginlay);
+            setListeners();
+    }
+
+
+
+    private void initViews() {
+
+
+    }
+
+    private void setListeners() {
+        loginButton.setOnClickListener(this);
+        signUp.setOnClickListener(this);
+    }
+
+    @Override
+    public void onClick(View v) {
+        switch (v.getId()) {
+            case R.id.login:
+                checkValidation();
+                break;
+
+            case R.id.sign:
+                final NavController navController = Navigation.findNavController(requireActivity(), R.id.my_nav_host_fragment);
+                final Bundle bundle = new Bundle();
+                navController.navigate(R.id.action_loginFragment_to_signupFragment, bundle);
+                break;
+        }
+    }
+    private void checkValidation() {
+        LoginUser();
+    }
+
+    private void LoginUser() {
+
+        String getEmailId = emailid.getText().toString();
```

Comment:
@Riyabelle25 Add a check here for the null values from the email and password edit texts. Currently, the app crashes if the login button is pressed with empty email and pw fields.